jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -130,6 +131,29 @@ export const usePipeline = (
     };
   }
 
+  const pollForIp = async (
+    project_name: string,
+    name: string,
+  ): Promise<string> => {
+    try {
+      const response = await requestAPI<PPS>(
+        `pps/${project_name}/${name}`,
+        'GET',
+      );
+      if ((response.details.service.ip !== null) && (response.details.service.ip.length > 0)) {
+        return `Create pipeline request successful. See output at ${response.details.service.ip}.`;
+      }
+    }
+    catch(e) {
+      if (e instanceof ServerConnection.ResponseError) {
+        setErrorMessage('Error retrieving pipeline: ' + e.response.statusText);
+      } else {
+        throw e;
+      }
+    }
+    return pollForIp(project_name, name);
```

Comment:
I don't know the fancy JS way to do this, but do we want to add a retry threshold?